packages/wonder-blocks-modal/components/modal-launcher/index.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+/**
+ * This component enables you to launch a modal, covering the screen.
+ *
+ * Children have access to `openModal` function via the function-as-children
+ * pattern, so one common use case is for this component to wrap a button:
+ *
+ * <ModalLauncher modal={<TwoColumnModal ... />}>
+ *     {({openModal}) => <button onClick={openModal}>Learn more</button>}
+ * </ModalLauncher>
+ *
+ * The actual modal itself is constructed separately, using a layout component
+ * like StandardModal, OneColumnModal, or TwoColumnModal, and is provided via
+ * the `modal` prop.
+ */
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import Color from "wonder-blocks-color";
+import {View} from "wonder-blocks-core";
+
+import ModalLauncherPortal from "./modal-launcher-portal.js";
+
+type Props = {
+    // The modal to render.
```

Comment:
The format of these comments aren't right and won't show up in styleguidist, you'll need to use `/** ... */` style comments